### ISSUE TITLE
Have `va-telephone` component render 711 numbers with TTY supporting text. (08.03.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#43969]
#### VFS team name
Debt Resolution

#### VFS product name
Combined VA Debt Portal

#### Point of Contact/Reviewers
Brian DeConinck (@briandeconinck) - Accessibility

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Button and link language isn't descriptive and/or their purpose isn't clear.

### Issue Details
Whenever a 711 number is provided, please include "TTY" in both the aria-label and the visible link text, eg. `<a href="tel:711" aria-label="TTY: 7 1 1.">TTY: 771</a>`. This helps users who are deaf or hard of hearing to quickly identify the number, and helps indicate to other users that they don't need to call that number (particularly screen reader users, who may be tabbing from link to link and might not hear the "If you have hearing loss" context).

### VA.gov Experience Standard
[Category Number 08, Issue Number 03](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 2.4.4_A

---

### Platform Recommendation
This is a known issue on the `va-telephone` component, [vets-design-system-documentation#775](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/775). Flagging here for DST visibility.

### VFS Team Tasks to Complete

- [ ] Comment on the ticket if there are questions or concerns
- [ ] Close the ticket when the issue has been resolved or validated by your Product Owner. If a team has additional questions or needs Platform help validating the issue, please comment in the ticket.

**Comment by k80bowman** [comment]
Thank you @briandeconinck! This is very helpful.

**Comment by bkjohnson** [comment]
This looks like it may be a duplicate of https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/775

**Comment by shiragoodman** [comment]
@bkjohnson re: your duplicate comment above... it very well might be, however this ticket was opened in the Collaboration Cycle Staging Review and is specific to an issue found with the product specified above. We ask that VFS teams (or Design System) don't close these tickets until we know the issue is resolved. Apologies for the duplication, but it's our way of tracking which issues were identified within the various VA.gov products. LMK if you have any other questions.

**Comment by bkjohnson** [comment]
I apologize - I didn't mean for this to be closed just yet. I'd like to announce this new feature and then it will be up to teams to use it, like `<va-telephone tty contact="711" />`.

**Comment by bkjohnson** [comment]
This new feature has [been announced in Slack](https://dsva.slack.com/archives/CE4304QPK/p1659031802724049).
